docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -242,22 +221,6 @@ data:
     images_rbd_glance_copy_timeout=600
     rbd_user=openstack
     rbd_secret_uuid=$CEPH_FSID
----
```

Comment:
ya so this is exactly what i would have done too.

we should not need to use customer service defieitons of nova or nova with cpeh anymore

we just need to use a single 

nova-extra-config configmap which we update as required in the procedure.

the only expction to that is if you need multiple ndoes sets with different nova config per nodeset in that case we will need a separate nova service per nodeset but +1 on this apporoch in general.